File: UtilidadesAPI_E/src/models/CompanyUser.js
const { DataTypes } = require('sequelize');
const sequelize = require('../config/database').sequelize;

const CompanyUser = sequelize.define('company_users', {
  id: {
    type: DataTypes.INTEGER,
    autoIncrement: true,
    primaryKey: true
  },
  id_company: {
    type: DataTypes.INTEGER,
    references: {
      model: 'company',
      key: 'id'
    }
  },
  companymemberships: {
    type: DataTypes.STRING,
    allowNull: false,
    defaultValue: ''
  },
  email: {
    type: DataTypes.STRING(128),
    allowNull: false,
    defaultValue: ''
  },
  isblocked: {
    type: DataTypes.INTEGER
  },
  languagecode: {
    type: DataTypes.STRING(128),
    allowNull: false,
    defaultValue: ''
  },
  name: {
    type: DataTypes.STRING(128),
    allowNull: false,
    defaultValue: ''
  },
  systemmemberships: {
    type: DataTypes.STRING,
    allowNull: false,
    defaultValue: ''
  },
  telephone: {
    type: DataTypes.STRING(128),
    allowNull: false,
    defaultValue: ''
  },
  timezone: {
    type: DataTypes.STRING(128),
    allowNull: false,
    defaultValue: ''
  },
  virtualoperatormemberships: {
    type: DataTypes.STRING,
    allowNull: false,
    defaultValue: ''
  }
  // Otros campos...
}, {
  tableName: 'company_users',
  timestamps: false, // Desactiva los timestamps
});

/* CompanyUser.associate = models => {
    CompanyUser.belongsTo(models.Company, {
    foreignKey: 'id_company',
    as: 'company'
  });
}; */

module.exports = CompanyUser;

File: UtilidadesDB_DataAccess/Models/CompanyUser.cs
using System.Text.Json.Serialization;

namespace UtilidadesDB_DataAccess.Models
{
    public partial class CompanyUser
    {
        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
        public int Id { get; set; }
        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
        public int IdCompany { get; set; }
        public bool IsBlocked { get; set; }
        public string SystemMemberships { get; set; }
        public string VirtualOperatorMemberships { get; set; }
        public string CompanyMemberships { get; set; }
        public string Timezone { get; set; }
        public string LanguageCode { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public string Telephone { get; set; }

        public virtual Company IdCompanyNavigation { get; set; }
    }
}
